app/pathogen/arbovirus/dashboard/(visualizations)/median-seroprevalence-by-who-region-and-age-group-table.tsx

```diff
@@ -0,0 +1,191 @@
+import { useContext, useMemo, useState } from "react";
+
+import { ArboContext } from "@/contexts/arbo-context/arbo-context";
+import {
+  mixColours,
+  typedGroupBy,
+  typedObjectEntries,
+  typedObjectFromEntries,
+  typedObjectKeys,
+} from "@/lib/utils";
+import { arbovirusesSF, convertArboSFtoArbo, median } from "./recharts";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import { WHORegion } from "@/lib/who-regions";
+import { Button } from "@/components/ui/button";
+
+enum AgeGroup {
+  "Adults (18-64 years)" = "Adults (18-64 years)",
+  "Children and Youth (0-17 years)" = "Children and Youth (0-17 years)",
+  "Seniors (65+ years)" = "Seniors (65+ years)",
+  "Multiple groups" = "Multiple groups",
+}
```

Comment:
Yeah I know there's a type in `recharts.tsx`, it's just I can do an `Object.values` on a union type.